addons/sys_profile/fnc_profileSimulator.sqf

```diff
@@ -201,8 +201,13 @@ if (!_simAttacks) then {
                                 if (!isnil "_profilePosition" && {!(_profilePosition isEqualTo [])} && {!isnil "_destination"}) then {
 
                                     // add z-index since some profiles dont one defined
-                                    _profilePosition set [2,0];
-                                    _destination set [2,0];
+                                    if ((count _profilePosition) == 2) then {
+                                        _profilePosition pushBack 0;
+                                    };
+
+                                    if ((count _destination) == 2) then {
+                                        _destination pushBack 0;
+                                    };
```

Comment:
I (personally) feel like in this case the `_variable set [2, 0];` is a bit clearer. `pushBack` kinda implies that something is being added to an array, which is true but also not _really_.